### ISSUE TITLE
The image crop screen is not WCAG accessibility compatible

### ISSUE THREAD

**Issue by SivaRamaB** [Yalantis/uCrop#719]
We are using this library crop functionality in  our app and it needs to comply with WCAG rules.

**What is the current behavior?**
Present behaviour is that the texts, controls, buttons are not WCAG compatible for both screen reader and keyboard. 
Icon controls in the camera edit photo screen are missing accessible names.
 
Custom controls in the camera edit photo views are not keyboard or screen reader operable.
Controls, like the photo rotation slider, are not keyboard or screen reader operable. 
They have no name role value or state spoken to TalkBack. 
![image-2020-10-24-12-28-07-153](https://user-images.githubusercontent.com/60690374/98626979-0f326100-2339-11eb-8b90-ce94cfc052cb.png)

Icon buttons in the camera edit photo screens are not meaningful or specific for screen reader users.
The X button incorrectly says "Navigate Up" when it actually closes the edit photo screen.
The checkmark icon button incorrectly says "Crop" when it is also the button the user needs to activate to be done with the screen or to be completed. The word "Crop" is confusing and does not help the screen reader user know the purpose of the control or what it does. 
![image-2020-10-24-12-35-49-998](https://user-images.githubusercontent.com/60690374/98627172-9253b700-2339-11eb-9d5f-ea336a413517.png)
![Screenshot_20201109-201527](https://user-images.githubusercontent.com/60690374/98627207-a39cc380-2339-11eb-8752-4ce5f4c80d07.png)

**What is the expected behavior?**
We need all the text, controls and button to be WCAG compatible for both screen reader and keyboard.
Make sure all icon controls have a meaningful accessible name spoken to the screen reader user.
Make sure all touch operable controls are keyboard and screen reader operable as well.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
We can reproduce these by using the Talkback feature in android devices.

**Which versions of uCrop, and which Android API versions are affected by this issue? Did this work in previous versions of uCrop?**
We are using "com.github.yalantis:ucrop:2.2.6" with Android target SDK 29

We need the WCAG success criteria as below,
4.1 Compatible - 4.1.2 Name, Role, Value (A) - Ensure that Assistive Technologies can gather information about, activate and keep up to date on the status of user interface controls in the content. 

![Screenshot_20201109-200053](https://user-images.githubusercontent.com/60690374/98626714-510ed780-2338-11eb-9c0b-93f8e3d5b0e6.png)
![Screenshot_20201109-200105](https://user-images.githubusercontent.com/60690374/98626722-553af500-2338-11eb-91cc-c46111fbbe6b.png)
![Screenshot_20201109-200133](https://user-images.githubusercontent.com/60690374/98626726-5835e580-2338-11eb-89a9-b6af30a73940.png)
![Screenshot_20201109-200111](https://user-images.githubusercontent.com/60690374/98626729-5a983f80-2338-11eb-9683-4062bfb551fb.png)
![Screenshot_20201109-200502](https://user-images.githubusercontent.com/60690374/98626731-5c620300-2338-11eb-9622-57524ed5c5a9.png)
